templates/haproxy.cfg.j2

```diff
@@ -37,15 +37,26 @@ defaults
 
 frontend {{ haproxy_frontend_name }}
     bind {{ haproxy_frontend_bind_address }}:{{ haproxy_frontend_port }}
+    {% if haproxy_ssl_pem %}
+    bind {{ haproxy_frontend_bind_address }}:{{ haproxy_frontend_ssl_port }} ssl crt /etc/haproxy/server.pem
```

Comment:
Does `/etc/haproxy/server.pem` need to be configurable? Does it work cross-platform (e.g. RHEL and Ubuntu)?